systemvm/js/ajaxviewer.js

```diff
@@ -184,7 +205,7 @@ KeyboardMapper.prototype = {
 			if(eventType == AjaxViewer.KEY_UP && (code == AjaxViewer.JS_KEY_ALT || code == AjaxViewer.JS_KEY_CTRL))
 				this.mappedInput.push({type : eventType, code: this.jsX11KeysymMap[code], modifiers: modifiers});
 			
-		} else if(eventType == AjaxViewer.KEY_PRESS && guestos == 'null') {
+		} else if(eventType == AjaxViewer.KEY_PRESS && guestos.toLowerCase() != 'windows') {
```

Comment:
Plan is to remove all OS strings and don't use them specifically at all.

I have left it here like this as removing this may break existing key mappings.  Getting key mappings right is time consuming and this code was written to get them right.